### DISCUSSION TITLE
de-duplicate multiple DAST tool records

### DISCUSSION THREAD

**Discussion by appsecsanta** [DefectDojo/django-DefectDojo#5844] [Q&A]
I want to match different dast tool results and group them under the same issue.  
(something like Found by: Acunetix, Netsparker and OWASP ZAP)

Is it possible with the legacy deduplication algorithm? What do you think?

I have tried updating settings.dist.py with DEDUPE_ALGO_LEGACY for the related tool parser, but no success so far.

**Comment by damiencarol** [comment]
@appsecsanta sadly, currently we don't support de-duplication between different tools.
The problem is, sometime the data are not comparable. So it become impossible to compare data from different tool.
We are working on improving the "reference" part. Maybe it will become possible in the near future to compare data with the same reference/data. (like for ex: CVE)

**Reply by appsecsanta** [reply]
I see, I was assuming this would be enough ['title', 'cwe', 'line', 'file_path', 'description']. 
Let me share my experience after testing with multiple tools.
